### PR TITLE
Eliane

### PR DESCRIPTION
**Pull request by sixolet**
Feedback, slow beatings, highs and lows

This drone is seven sines phase-modulating each other in a ring.

The sines are based around a natural minor scale (ET), jumping by a fifth
or a diminished fifth each to the next. And each sine is phase modulated
by the next sine, and the one after that, until the highest sines are phase
modulated by the lowest.

There's a lot of different sounds in here and it changes slowly; give it some patience.

### CODE REVIEW

File: engine/drones/Eliane.scd [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// (seven sines phase-modulating each other in a crosslinked ring)
+
+{ 
+  arg hz=220,amp=1.0;
+	var base = 2.pow(1/12.0);
+	// All the notes in a natural minor scale, but by fifths (with one diminished).
+	// root, fifth, second + 8v, minor sixth +8v, minor third +2 8v, minor seventh +2 8v, fourth +3 8v 
+	var intervals = base.pow([0, 7, 14, 20, 27, 34, 41]);
+	var busses = LocalIn.ar(14);
+	var nice = LFNoise2.kr(200.reciprocal);
```
Comment (ryanlaws) [inline engine/drones/Eliane.scd:11]: `1/12.0` and `200.reciprocal`, eh? I like your style 😎

File: engine/drones/Eliane.scd [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	var bigsound;
+	var sines = 7.collect { |x|
+		var next = (x+1)%7;
+		var nextayim = (x+2)%7;
+		var modulator1 = [busses[2*next], busses[2*next+1]];
```
Comment (ryanlaws) [inline engine/drones/Eliane.scd]: Genuine question - is this
`[busses[2*next], busses[2*next+1]]`
different from this?
`busses[2*next + [0,1]]`
Actually now that I typed it out, it's kinda ugly, isn't it.

File: engine/drones/Eliane.scd [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		var stereo;
+		var pitch = hz*intervals[x];
+		mod = pi * LFNoise2.kr(0.015).unipolar().pow(nice.range(0.9, 2)) * mod;
+		stereo = SinOsc.ar(pitch + ((x/2)*LFNoise2.kr(0.1!2)), mod);
+		// At high pitch, introduce subharmonics instead of high intervals.
+		switch (x)
+		{6} {
+			stereo = SelectX.ar(hz/(150+hz), [
+				stereo/((1.5*x)+1),
+				SinOsc.ar(hz/2 + LFNoise2.kr(0.1!2), mod),
```
Comment (ryanlaws) [inline engine/drones/Eliane.scd]: I love that `Noise(amp!2)` trick 🎨

File: engine/drones/Eliane.scd [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+{ 
+  arg hz=220,amp=1.0;
+	var base = 2.pow(1/12.0);
+	// All the notes in a natural minor scale, but by fifths (with one diminished).
+	// root, fifth, second + 8v, minor sixth +8v, minor third +2 8v, minor seventh +2 8v, fourth +3 8v 
+	var intervals = base.pow([0, 7, 14, 20, 27, 34, 41]);
+	var busses = LocalIn.ar(14);
+	var nice = LFNoise2.kr(200.reciprocal);
+	var bigsound;
```
Comment (ryanlaws) [inline engine/drones/Eliane.scd]: Variable name of the year 🏅

**Review by ryanlaws — APPROVED** [review]

File: engine/drones/Eliane.scd (hunk 2 above)
Comment (sixolet) [reply engine/drones/Eliane.scd]: I have sorted this using another reshape

**Comment by tyleretters** [comment]
@ryanlaws @sixolet ok to merge?

**Comment by sixolet** [comment]
Encouraged by me. I think I’m done tweaking, it’s sounding good.

**Pull request merged**